File: 4-Tarea-Condicionales-Ciclos/A.DFD-Aprobatoria-o-Reprobatoria.java
import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    double calificacion = scanner.nextDouble();

    if (calificacion < 6.0)
      System.out.println("Reprobatoria");
    else
      System.out.println("Aprobatoria");
  }
}

File: 4-Tarea-Condicionales-Ciclos/A.DFD-Aprobatoria-o-Reprobatoria.cpp
#include <iostream>
using namespace std;

int main() {
  double calificacion;
  cin >> calificacion;

  if (calificacion < 6.0)
    cout << "Reprobatoria" << endl;
  else
    cout << "Aprobatoria" << endl;

  return 0;
}